specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+list) is compared with the list of languages for which the user has declared a preference (the user
+profile language list). The app runtime language list is set to this intersection (if the
+intersection is not empty), or to just the app's default language (if the intersection is empty).
+For more detail, see the
+[App runtime language list](https://learn.microsoft.com/windows/apps/design/globalizing/manage-language-and-region#app-runtime-language-list)
+section in
+[Understand user profile languages and app manifest languages](https://learn.microsoft.com/windows/apps/design/globalizing/manage-language-and-region).
+
+## ApplicationLanguages.ManifestLanguages property
+
+Gets the app's declared list of supported languages.
+
+# Property Value
+
+[IReadOnlyList](https://learn.microsoft.com/dotnet/api/system.collections.generic.ireadonlylist-1)<[String](https://learn.microsoft.com/dotnet/api/system.string)>
+
+The list of supported languages declared in the app's manifest. For un-packaged applications,
+exception is thrown.
+
+Note: This property returns the same values as the language list exposed by
+[Windows.Globalization.ApplicationLanguages.ManifestLanguages](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.manifestlanguages)
+property.
+
+# Remarks
+
+When language resource packages are used, the packages that are installed and registered for a given
+user are determined by the languages in the user's language profile. The set of languages returned
+by the ManifestLanguages property is limited to the languages currently available on the system for
+the user. Languages included in the main app package manifest will always be returned; languages
+from resource packages will be returned only if the language is applicable for the user (that is, is
+in the user's preferences) and the resource package has been installed and registered for the user
+at the time the property is accessed.
+
+## ApplicationLanguages.PrimaryLanguageOverride property
+
+Gets or sets an override for the app's preferred language, expressed as a
+[BCP-47 language tag](https://tools.ietf.org/html/bcp47). For packaged apps, this property is a
+wrapper for
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property and behaves the same. This setting is global for the running process. Unlike for packaged
+apps, for un-packaged apps, this setting is not persisted between app sessions.
+
+```c#
+public static string PrimaryLanguageOverride { get; set; }
+```
+
+# Property Value
+
+[String](https://learn.microsoft.com/dotnet/api/system.string)
+
+A [BCP-47 language tag](https://tools.ietf.org/html/bcp47). The app can set it to override the
+language. It must be a single language tag; a delimited list of language tags will result in default
```

Comment:
Does "default language" mean "it takes the first language in your list", "the property keeps the value it had before you tried to set it to a list", or "the property gets reset to empty by this attempt"?

Reply:
Actually, this is wrong. Setting delimited list of language tags will result in "The parameter is incorrect." as well. This is copy-pasted from W.G. property docs, so the docs are wrong there. This line needs to be fixed.

Reply:
RECOMMEND: Update this doc to state that it throws exception. 

CONSIDER: Update existing public doc.